README.md

```diff
@@ -6,22 +6,20 @@ Notes for programming:
 * Setup Arduino: https://www.hackster.io/john-bradnam/using-the-new-attiny-processors-with-arduino-ide-612185
 * You will need to use the megaTinyCore library by SpenceKonde: https://github.com/SpenceKonde/megaTinyCore
 
-![ArduinoConfig](https://user-images.githubusercontent.com/31858409/110745376-0f813100-8233-11eb-9dcd-ba6e8b78217b.PNG)
-
 Upload settings can be found in the arduino program. 
 
 Please note that there are a number of ATTiny SKUs that are compatible with this PCB and code, please refer to the below table when choosing an appropriate IC.
 
 | Model number   | Confirmed   | Notes                             |
 | -------------- | ----------- | --------------------------------- |
-| ATtiny204SSN_, | Yes         | Support SerialEvent must be 'no'  |
+| ATtiny204SSN_, | Yes         | See Below(1)  |
 | ATtiny404SSN_, | Yes         |    	                           |
 | ATtiny804SSN_, | Yes         |                                   |
 | ATtiny1604SSN_,| Yes         |    	                           |
-| ATtiny214SSN_, | No          | Support SerialEvent must be 'no'  |
-| ATtiny414SSN_, | No          |    	                           |
-| ATtiny814SSN_, | No          |                                   |
-| ATtiny1614SSN_,| No          |    	                           |
-| ATtiny1624SSU, | No          |    	                           |
+| ATtiny214SSN_, | Yes         | See Below(1)  |
+| ATtiny414SSN_, | Yes         |    	                           |
+| ATtiny814SSN_, | Yes         |                                   |
+| ATtiny1614SSN_,| Yes         |    	                           |
+| ATtiny1624SSU, | Yes         |    	                           |
 
-ATtiny 204 and 214 must have all serial debug removed in addition to the startup sweeping LED sequence to fit in the 2K code space.
+(1)ATtiny 204 and 214 have 2K flash memory for program space. The preprocessor for TWOKFLASHCHIP must be uncommented prior to compiling. See Line 9. 
```

Comment:
Probably worth having at least some of this in the pcb-firmware repo?

Reply:
@darraghbr READMEs have been adjusted.